html/pages/apps/proxmox/vm.inc.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+global $vars;
```

Comment:
This line can be removed, using `global` outside a function context doesn't really do anything useful